db/migrate/20131017150735_create_automatic_exploitation_match_results.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAutomaticExploitationMatchResults < ActiveRecord::Migration
+  def change
+    create_table :automatic_exploitation_match_results do |t|
+      t.integer :match_id
```

Comment:
Foreign keys should use `t.references` and be marked `null: false` when required.